src/_includes/_layouts/ForStudents.jsx

```diff
@@ -20,6 +24,213 @@ export default ({ comp, faqs }) => (
         </h1>
       </section>
       <comp.StudentsProjectTeam comp={comp} />
+      {/* Application Process Section */}
```

Comment:
Are you able to do a small refactor? We want to break each page seciton into its own individual page component. So ApplicationProcess.jsx and InternalDesignTeam.jsx

Reply:
Already done